app/models/spree_avatax/sales_shared.rb

```diff
@@ -173,6 +173,10 @@ def gettax_params(order, doc_type)
       }
     end
 
+    def customer_code(order)
+      order.user ? order.user_id : REXML::Text.normalize(order.email)[0, 50]
```

Comment:
Why do we have to use `REXML::Text`?

Reply:
Thats what the code was using before, I think it's just to be safe and escape any unsafe characters in the email

Reply:
Ah gotcha!